rho/postprocessing.py

```diff
@@ -766,6 +770,146 @@ def process_fuse_init_files(fact_names, host_vars):
                 '; '.join(chkconfig_out['stdout_lines']))}
 
 
+def classify_kie_file(pathname):
+    """Classify a kie-api-* file.
+
+    :param pathname: the path to the file
+    :returns: a BRMS version string, or None if not a Red Hat kie file..
+    """
+
+    # os.path.basename behaves differently if the last part of the
+    # path ends in a /, so normalize.
+    if pathname.endswith('/'):
+        pathname = pathname[:-1]
+
+    basename = os.path.basename(pathname)
+
+    version_string = utilities.strip_suffix(
+        utilities.strip_prefix(basename, 'kie-api-'),
+        '.jar')
+
+    if version_string in BRMS_CLASSIFICATIONS:
+        return BRMS_CLASSIFICATIONS[version_string]
+
+    if 'redhat' in version_string:
+        return version_string
+
+    return None
+
+
+JBOSS_BRMS = 'jboss.brms'
```

Comment:
What about moving the constants to the top of the file? After the import blocks?

Reply:
That's not a style I'm used to, but I'm happy to adopt a new one. However, I don't want to do it through individual comments on pull requests. I think those have a tendency to be incomplete, and it's hard to be sure that everyone has the same idea of what "quipu style" is.

I would fully support you writing a proposed style guide that we can talk about and agree to. If we had a team style guide, I would be happy to follow it in my code and start enforcing it in code reviews. It might be easier to start with https://google.github.io/styleguide/pyguide.html and then make a list of differences between that and the style you want us to use.